conans/test/functional/toolchains/google/test_bazel.py

```diff
@@ -1,125 +1,251 @@
-import os
 import platform
 import textwrap
-import unittest
 
 import pytest
 
-from conans.test.assets.sources import gen_function_cpp
 from conans.test.utils.tools import TestClient
 
 
-@pytest.mark.tool_bazel
-class Base(unittest.TestCase):
-
-    conanfile = textwrap.dedent("""
-        from conan.tools.google import Bazel
-        from conans import ConanFile
-
-        class App(ConanFile):
-            name="test_bazel_app"
-            version="0.0"
-            settings = "os", "compiler", "build_type", "arch"
-            generators = "BazelDeps", "BazelToolchain"
-            exports_sources = "WORKSPACE", "app/*"
-            requires = "hello/0.1"
-
-            def layout(self):
-                self.folders.build = "bazel-build"
-                self.folders.generators = "bazel-build/conandeps"
-
-            def build(self):
-                bazel = Bazel(self)
-                bazel.configure()
-                bazel.build(label="//app:main")
-
-            def package(self):
-                self.copy('*', src='bazel-bin')
+@pytest.fixture(scope="module")
+def bazelrc():
+    return textwrap.dedent("""
+        build:Debug -c dbg --copt=-g
```

Comment:
Isn't it a bit weird that this is not part of the ``conan new`` template? why not?

Reply:
Ok, It might be useful but take into account, that is just an "invented" configuration file for bazel, it is very likely that a bazel user use different values, the "pre-cooked" build types in bazel are "opt", "dbg" and "fastbuild". Also the `--color` and the `withTimeStamps` are totally useless and just make sense in the test.

Reply:
Do we generate in the "conan new" also some base profiles (Release, Debug) setting the config name matching the bazelrc?